jsonnet/thanos-ruler.jsonnet

```diff
@@ -28,22 +28,21 @@ local authorizationRole =
 {
   local thanosRulerName = 'user-workload',
   local thanosRulerConfig = super._config +
-  {
-    name: thanosRulerName,
-    namespace: 'openshift-user-workload-monitoring',
-    labels: {
-      'app.kubernetes.io/name': thanosRulerName,
-    },
-    selectorLabels: {
-      'app': 'thanos-ruler',
-      'thanos-ruler': thanosRulerName,
-    },
-    ports: {
-      grpc: 10901,
-      http: 10902,
-    },
-
-  },
+                            {
+                              name: thanosRulerName,
+                              namespace: 'openshift-user-workload-monitoring',
+                              labels: {
+                                'app.kubernetes.io/name': thanosRulerName,
+                              },
+                              selectorLabels: {
+                                app: 'thanos-ruler',
+                                'thanos-ruler': thanosRulerName,
+                              },
+                              ports: {
+                                web: 9091,
+                                tenancy: 9092,
+                              },
+                            },
```

Comment:
It seems formatting is off.

Reply:
this is my local `jsonnetfmt` (v0.14.0)...

Reply:
v0.15.0 is out, just fyi, but else lgtm if jsonnetfmt causes this :+1: